packages/app-store/hubspot/lib/CrmService.ts

```diff
@@ -227,7 +229,7 @@ export default class HubspotCalendarService implements CRM {
     return contacts.map((contact) => {
       return {
         id: contact.id,
-        email: contact.properties.email,
+        email: contact.properties.email || "",
```

Comment:
Emails are required for contacts so not sure why it's throwing a `string | null` type here.